docs/design/persistentdisks.md

```diff
@@ -0,0 +1,199 @@
+# Persistent Storage
+
+
+## Abstract
+
+This document defines persistent, cluster-scoped storage for applications requiring long lived data.  The term "PersistentStorage" was used to avoid overloading the term "Volume."  Storage is exposed to a pod as a volume.  Developers and pod authors request storage by defining their performance needs and are matched with an appropriate object from a pool maintained by administrators.
```

Comment:
Since the point of storage is to be used as a Volume, I'd still prefer PersistentVolume.